src/components/layout.tsx

```diff
@@ -6,15 +6,15 @@
  */
 
 import { graphql, useStaticQuery } from 'gatsby'
-import React from 'react'
+import React, { ReactElement, ReactNode } from 'react'
 
 import { Theme } from '../theme'
 
 interface Props {
-  children?: any
+  children?: ReactNode
```

Comment:
Je použito množné číslo children, ale přitom se používá "single" typ (namísto třeba `ReactNode[]`). Je to tak správně?

Reply:
> Je použito množné číslo children, ale přitom se používá "single" typ (namísto třeba ReactNode[]). Je to tak správně?

jojo, to je je v pohode. `children` je Reactom vyhrazeny keyword, které je vždy `ReactNode`, sudé `ReactNode[]` je `ReactNode ` :)